charts/internal/machine-class/templates/machine-class.yaml

```diff
@@ -55,6 +55,10 @@ providerSpec:
   dnsConfig:
 {{ toYaml $machineClass.dnsConfig | indent 4 }}
 {{- end }}
+{{- if $machineClass.memoryFeatures }}
+  memoryFeatures:
```

Comment:
Could we rename this to just `memory` instead of `memoryFeatures` (everywhere), same as in KubeVirt? Or do you see any strong reason to use a different name?

Reply:
OK, I figured out the strong reason, `memory` is already in use for the memory size, let's have `memoryFeatures` then.

Reply:
yes